src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -1,10 +1,9 @@
 package frc.robot.constants;
 
-import edu.wpi.first.math.controller.SimpleMotorFeedforward;
 import edu.wpi.first.math.util.Units;
 
 public final class IntakeConstants {
-    public static final String LEFT_SPARK = "intake spark 1";
+    public static final String TALON = "intake spark 1";
     public static final int LEFT_SPARK_ID = 1;
     public static final String RIGHT_SPARK = "intake spark 2";
     public static final int RIGHT_SPARK_ID = 2;
```

Comment:
1. Rename it from intake spark 1 to just intake falcon
2. Remove the Right Spark
3. Change the `Left_Spark_ID` to Intake Falcon ID and set it to 1
4. Remove `RIGHT_SPARK_ID.